mathesar_ui/src/component-library/dropdown/Dropdown.svelte

```diff
@@ -40,6 +40,13 @@
   $: tgClasses = calculateTriggerClass(triggerClass, showArrow);
 
   function toggle(e: Event) {
+    /**
+     * To make it work when wrapped with an anchor tag.
+     * Since the event is already not bubbling up
+     * and this component will always necessarily handle the click event,
+     * preventing the default behaviour should be fine
+     */
+    e.preventDefault();
```

Comment:
This is a hack that's not great for accessibility. Ideally we shouldn't ever have a `<button>` element within an `<a>` element because that's invalid HTML. I opened #1840, proposing an alternate approach. Working on that ticket is low priority, but this concept would be good to keep in-mind for new code you write going forward.